src/chainstate/stacks/db/blocks.rs

```diff
@@ -122,7 +122,6 @@ pub struct StagingMicroblock {
 #[derive(Debug, Clone, PartialEq)]
 pub struct StagingBlock {
     pub consensus_hash: ConsensusHash,
-    pub burn_header_timestamp: u64,
```

Comment:
Why are we getting rid of `burn_header_timestamp`?

Reply:
We aren't.  It's just not necessary to store it in the `staging_blocks`.  We only need to store it in the `headers`.

Reply:
Got it, thank you!